libc/newhdrgen/yaml/math.yaml

```diff
@@ -2199,7 +2206,44 @@ functions:
       - type: float128
       - type: int
       - type: unsigned int
-    guard: LIBC_TYPES_HAS_FLOAT128
+    guard: LIBC_TYPES_HAS_FLOAT128scalbln
+  - name: scalbln
+    standards: 
+      - stdc
+    return_type: double
+    arguments:
+      - type: double
+      - type: long
+  - name: scalblnl
+    standards:
+      - stdc
+    return_type: long double
+    arguments:
+      - type: long double
+      - type: long
+  - name: scalblnf
+    standards:
+      - stdc
+    return_type: long double
+    arguments:
+      - type: float
+      - type: long
+  - name: scalblnf16
+    standards:
+      - stdc
+    return_type: float16
+    arguments:
+      - type: float16
+      - type: long
+    guard: LIBC_HAS_TYPES_FLOAT16
+  - name: scalblnf128
+    standards:
+      - llvm_libc_ext
```

Comment:
All the `scalbln*` functions are standard C functions. They're not LLVM libc extensions.

You can use https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3096.pdf if you need access to a draft of the C23 standard. You can also look up pre-C23 functions on cppreference.com.